contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/hbm/BaseReviewServiceImpl.java

```diff
@@ -316,7 +363,29 @@ public void processQueue() {
 		
 	}
 
+	public ContentReviewItem getItemBySubmissionId(String submissionId){
+		Search search = new Search();
+		search.addRestriction(new Restriction("submissionId", submissionId));
```

Comment:
Not sure what you mean, I need to scan the table for getting the item as this wasn't previously implemented